### PR TITLE
[zh]: Rsyc content/en/docs/setup/production-environment/tools/kubeadm  setup-1

### PR DESCRIPTION
**Pull request by zaunist**
#33475  setup-1

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 1]
```diff
@@ -122,22 +122,13 @@ sudo kubeadm reset
...
-journalctl -ul docker
-```
+A possible solution is to restart the container runtime and then re-run `kubeadm reset`.
+You can also use `crictl` to debug the state of the container runtime. See
+[Debugging Kubernetes nodes with crictl](/zh/docs/tasks/debug-application-cluster/crictl/).
 -->
 ## 当删除托管容器时 kubeadm 阻塞
 
-如果 Docker 停止并且不删除 Kubernetes 所管理的所有容器，可能发生以下情况：
+如果容器运行时停止并且不删除 Kubernetes 所管理的所有容器，可能发生以下情况：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:131]:
```suggestion
如果容器运行时停止并未删除 Kubernetes 所管理的所有容器，可能发生以下情况：
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 2]
```diff
@@ -152,17 +143,8 @@ sudo kubeadm reset
...
-sudo kubeadm reset
-```
-
-检查 docker 的日志也可能有用：
-
-```shell
-journalctl -ul docker
-```
+你也可以使用`crictl`来调试容器运行时的状态。参见
+[Debugging Kubernetes nodes with crictl](/zh/docs/tasks/debug-application-cluster/crictl/)
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:147]:
```suggestion
你也可以使用`crictl`来调试容器运行时的状态。
参见[使用 CRICTL 调试 Kubernetes 节点](/zh/docs/tasks/debug/debug-cluster/crictl/)。
```

**Comment by tengqm** [comment]
I'd suggest combining commits to all three files under 'setup-1' into this PR.

**Comment by zaunist** [comment]
> I'd suggest combining commits to all three files under 'setup-1' into this PR.

Got it, I will merge this commit later

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 3]
```diff
@@ -130,94 +128,111 @@ nc 127.0.0.1 6443
...
-如果你不指定运行时，则 kubeadm 会自动尝试检测到系统上已经安装的运行时，
-方法是扫描一组众所周知的 Unix 域套接字。
-下面的表格列举了一些 kubeadm 查找的容器运行时及其对应的套接字路径：
-
-| 运行时     | 域套接字                         |
-|------------|----------------------------------|
-| Docker Engine  | `/var/run/dockershim.sock`        |
-| containerd     | `/run/containerd/containerd.sock` |
-| CRI-O          | `/var/run/crio/crio.sock`         |
+如果你不指定运行时，kubeadm 会自动尝试通过扫描已知的端点列表来检测已安装的容器运行时
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:161]:
```suggestion
如果你不指定运行时，kubeadm 会自动尝试通过扫描已知的端点列表来检测已安装的容器运行时。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 4]
```diff
@@ -130,94 +128,111 @@ nc 127.0.0.1 6443
...
+{{< table >}}
+| Runtime                            | Path to Unix domain socket                   |
+|------------------------------------|----------------------------------------------|
+| containerd                         | `unix:///var/run/containerd/containerd.sock` |
+| CRI-O                              | `unix:///var/run/crio/crio.sock`             |
+| Docker Engine (using cri-dockerd)  | `unix:///var/run/cri-dockerd.sock`           |
+{{< /table >}}
+-->
+{{< table >}}
+| 运行时                              | 域套接字                                     |
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:211]:
```suggestion
| 运行时                              |  UNIX 域套接字                                     |
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 5]
```diff
@@ -130,94 +128,111 @@ nc 127.0.0.1 6443
...
+| Runtime                            | Path to Windows named pipe                   |
+|------------------------------------|----------------------------------------------|
+| containerd                         | `npipe:////./pipe/containerd-containerd`     |
+| Docker Engine (using cri-dockerd)  | `npipe:////./pipe/cri-dockerd`               |
+{{< /table >}}
+-->
+{{% tab name="Windows" %}}
+
+{{< table >}}
+| 运行时                              | 域套接字                                     |
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:231]:
```suggestion
| 运行时                              |  Windows 命名管道路径                                     |
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 6]
```diff
@@ -22,46 +22,46 @@ or upgrades for such nodes. The long term plan is to empower the tool
 [etcdadm](https://github.com/kubernetes-sigs/etcdadm) to manage these
 aspects.
 -->
-在本指南中，当 kubeadm 用作为外部 etcd 节点管理工具，请注意 kubeadm 不计划支持此类节点的证书更换或升级。对于长期规划是使用 [etcdadm](https://github.com/kubernetes-sigs/etcdadm) 增强工具来管理这方面。
+在本指南中，当 kubeadm 用作为外部 etcd 节点管理工具，请注意 kubeadm 不计划支持此类节点的证书更换或升级。
+对于长期规划是使用 [etcdadm](https://github.com/kubernetes-sigs/etcdadm) 增强工具来管理这方面。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:26]:
```suggestion
对于长期规划是使用 [etcdadm](https://github.com/kubernetes-sigs/etcdadm) 增强工具来管理这些方面。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 7]
```diff
@@ -22,46 +22,46 @@ or upgrades for such nodes. The long term plan is to empower the tool
...
-when using kubeadm to set up a kubernetes cluster.
+By default, kubeadm runs a local etcd instance on each control plane node.
+It is also possible to treat the etcd cluster as external and provision
+etcd instances on separate hosts. The differences between the two approaches are covered in the
+[Options for Highly Available topology](/docs/setup/production-environment/tools/kubeadm/ha-topology) page.
 -->
-默认情况下，kubeadm 运行单成员的 etcd 集群，该集群由控制面节点上的 kubelet 以静态 Pod 的方式进行管理。由于 etcd 集群只包含一个成员且不能在任一成员不可用时保持运行，所以这不是一种高可用设置。本任务，将告诉你如何在使用 kubeadm 创建一个 kubernetes 集群时创建一个外部 etcd：有三个成员的高可用 etcd 集群。
 
+默认情况下，kubeadm 在每个控制平面节点上运行一个本地 etcd 实例。也可以使用外部的 etcd 集群，并在不同的主机上提供 etcd 实例。
+这两种方法的区别在 [Options for Highly Available topology](/docs/setup/production-environment/tools/kubeadm/ha-topology) 页面中有所阐述。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:37]:
```suggestion
这两种方法的区别在[高可用拓扑的选项](/zh/docs/setup/production-environment/tools/kubeadm/ha-topology)页面中阐述。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 8]
```diff
@@ -131,39 +145,52 @@ kubeadm 包含生成下述证书所需的所有必要的密码学工具；在这
    使用以下脚本为每个将要运行 etcd 成员的主机生成一个 kubeadm 配置文件。
 
    ```sh
-   # 使用 IP 或可解析的主机名替换 HOST0、HOST1 和 HOST2
+   # 使用你的主机 IP 更新 HOST0、HOST1 和 HOST2
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:148]:
```suggestion
   # 使用你的主机 IP 替换 HOST0、HOST1 和 HOST2 的 IP 地址
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md [hunk 9]
```diff
@@ -122,22 +122,13 @@ sudo kubeadm reset
...
-journalctl -ul docker
-```
+A possible solution is to restart the container runtime and then re-run `kubeadm reset`.
+You can also use `crictl` to debug the state of the container runtime. See
+[Debugging Kubernetes nodes with crictl](/zh/docs/tasks/debug-application-cluster/crictl/).
 -->
 ## 当删除托管容器时 kubeadm 阻塞
 
-如果 Docker 停止并且不删除 Kubernetes 所管理的所有容器，可能发生以下情况：
+如果容器运行时停止并未删除 Kubernetes 所管理的所有容器，可能发生以下情况：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md:131]:
```suggestion
如果容器运行时停止并且未删除 Kubernetes 所管理的容器，可能发生以下情况：
```

**Pull request merged**